tests/sqlparser_snowflake.rs

```diff
@@ -525,6 +525,307 @@ fn test_snowflake_single_line_tokenize() {
+                                            )),
+                                        }
+                                    )),
+                                    order: Some(IdentityOrder::Noorder),
+                                }
+                            ))
+                        }]
+                    },
+                    ColumnDef {
+                        name: "c".into(),
+                        data_type: DataType::Int(None),
+                        collation: None,
+                        options: vec![ColumnOptionDef {
+                            name: None,
+                            option: ColumnOption::Identity(Identity::Identity(IdentityProperty {
+                                parameters: None,
+                                order: None,
+                            }))
+                        }]
+                    },
+                    ColumnDef {
+                        name: "d".into(),
+                        data_type: DataType::Int(None),
+                        collation: None,
+                        options: vec![ColumnOptionDef {
+                            name: None,
+                            option: ColumnOption::Identity(Identity::Identity(IdentityProperty {
+                                #[cfg(not(feature = "bigdecimal"))]
+                                parameters: Some(IdentityFormat::StartAndIncrement(
+                                    IdentityParameters {
+                                        seed: Expr::Value(Value::Number("100".to_string(), false)),
+                                        increment: Expr::Value(Value::Number(
+                                            "1".to_string(),
+                                            false
+                                        )),
+                                    }
+                                )),
+                                #[cfg(feature = "bigdecimal")]
+                                parameters: Some(IdentityFormat::StartAndIncrement(
+                                    IdentityParameters {
+                                        seed: Expr::Value(Value::Number(
+                                            bigdecimal::BigDecimal::from(100),
+                                            false,
+                                        )),
+                                        increment: Expr::Value(Value::Number(
+                                            bigdecimal::BigDecimal::from(1),
+                                            false,
+                                        )),
+                                    }
+                                )),
+                                order: Some(IdentityOrder::Order),
+                            }))
+                        }]
+                    },
+                ]
+            );
+        }
+        _ => unreachable!(),
+    }
+}
+
+#[test]
+fn test_snowflake_create_table_with_collated_column() {
```

Comment:
for the tests, in addition to the single option ones like these can we add coverage for using multiple options (and a mixture of WITH and without the keyword where applicable)?

Reply:
I updated all existing tests with two variants of tested SQL - without and with `WITH`.
Additionally I added one more test with several column options in SQL to make sure that all these options are parsed together.